flake.nix

```diff
@@ -13,11 +12,6 @@
       url = "https://install.determinate.systems/determinate-nixd/tag/v3.4.2/x86_64-linux";
       flake = false;
     };
-    determinate-nixd-aarch64-darwin = {
```

Comment:
This input is used since we generate packages for all [supported systems](https://github.com/DeterminateSystems/determinate/blob/main/flake.nix#L25). We should check to make sure the package is still being used in this flake.

Reply:
Yeah, I'm setting this to a draft because I didn't fully appreciate the interwovenness of things here